api/genomics/docs/unsorted_vcf.md

```diff
@@ -0,0 +1,31 @@
+## Answering some questions on how our tools interact with unsorted VCF files
+
+### Does Plink require a VCF file to be sorted?
+No, but another flag `--make-bed` needs to be added to the binary file creation step.
+`plink --vcf-half-call m --const-fid 0 --vcf unsorted.vcf --out cohort --make-bed` ->  
+`plink --vcf-half-call m --const-fid 0 --vcf unsorted.vcf --out cohort --make-bed`
```

Comment:
What happens if you don't use `--make-bed` ? Does it error out?

(also, nice flag name)

Reply:
yeah. it will fail without the flag